xline/src/storage/kvstore.rs

```diff
@@ -37,11 +35,13 @@ pub(crate) struct KvStore {
     exec_tx: mpsc::Sender<ExecutionRequest>,
     /// Sender to send sync request
     sync_tx: mpsc::Sender<SyncRequest>,
+    /// KV watcher
+    kv_watcher: Arc<KvWatcher>,
 }
 
 /// KV store inner
 #[derive(Debug)]
-struct KvStoreInner {
+pub(crate) struct KvStoreInner {
```

Comment:
Why is the inner structure `pub(crate)`?

Reply:
This is referred by Kv_watcher. Will change the name.

Reply:
Done